cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,332 @@
+
+1.  Define a client PKI role in Vault:
+    
+    ~~~txt
+    vault write cockroach_client_ca/roles/client \
+    allow_any_name=true \
+    client_flag=true \
+    enforce_hostnames=false \
+    allow_ip_sans=false \
+    allow_localhost=false \
+    max_ttl=48h
+    ~~~
+
+1. Create PKI authentication credentials (a private key and public certificate) for a root user. 
+
+    {{site.data.alerts.callout_info}}
+    CockroachDB takes the name of the SQL user to be authenticated from the `common_name` field.
+    {{site.data.alerts.end}}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault write "cockroach_client_ca/issue/client" \
+    common_name=root \
+    --format=json > "${SECRETS_DIR}/clients/certs.json"
+    ~~~
+
+1. Parse the client key and certificate pair from the payload.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    echo -e $(cat "${SECRETS_DIR}/clients/certs.json" | jq .data.private_key | tr -d '"') > "${SECRETS_DIR}/clients/client.root.key"
+    echo -e $(cat "${SECRETS_DIR}/clients/certs.json" | jq .data.certificate | tr -d '"') > "${SECRETS_DIR}/clients/client.root.crt"
+    ~~~
+
+1. Set key file permissions; CockroachDB will reject improperly permissioned private keys for authentication.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    chmod 0600  ${SECRETS_DIR}/clients/client.root.key
+    chown $USER ${SECRETS_DIR}/clients/client.root.key
+    ~~~
+
+## Upload a certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster
+
+Add a CA certificate to your cluster's trust store for client authentication. Client certificates signed using the private key corresponding to this certificate will be accepted by your cluster for certificate-based client authentication.
+
+Refer to [Transport Layer Security (TLS) and Public Key Infrastructure (PKI): The CockroachDB certificate Trust Store](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html#the-cockroachdb-certificate-trust-store)
+
+{{site.data.alerts.callout_success}}
+Managing the certificate authority (CA) certificate for a {{ site.data.products.dedicated }} cluster requires the [Cluster Administrator](authorization.html#cluster-administrator) or [Org Administrator (legacy)](authorization.html#org-administrator-legacy) Organization role.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="api">Using the API</button>
+  <button class="filter-button page-level" data-scope="tf">Using Terraform</button>
+</div>
+<p></p>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+1. Submit the asynchronous request, supplying your cluster ID, API key, and the path to the certificate JSON with your CA certificate, as described in [Create the certificate authority (CA) certificate](#create-the-certificate-authority-ca-certificate).
```

Comment:
Indent the code block if it belongs with the step. (similar below)

It could be confusing to show the 200 response for an asynchronous request. Is it worth explaining that the 200 only indicates the success of the submission, not the success of the request?